src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -705,11 +704,10 @@ function DeleteRecord() {
 }
 
 CHAMELEON.CORE.MTTableEditor.DeleteRecordWithCustomConfirmMessage = function (sConfirmText) {
-    if (sCurrentRecordName != '') {
-        sConfirmText += "\n\n " + CHAMELEON.CORE.i18n.Translate('chameleon_system_core.js.record') + ": \"" + sCurrentRecordName + '"';
-    } else {
-        sConfirmText += "\n\n " + CHAMELEON.CORE.i18n.Translate('chameleon_system_core.js.record') + ": \"" + CHAMELEON.CORE.i18n.Translate('chameleon_system_core.js.unnamed') + '"';
-    }
+    var currentRecordName = $('#tableEditorContainer .navbar-brand').text();
```

Comment:
There is still a "sCurrentRecordName" here?

Reply:
sCurrentRecordName does not exist anymore. It was used to set the record name only for the delete method.
This was not necessary because we could get the name from the title of the record.
I added to update the title on save so this fixes a very old bug, two, where a changed name never updated the title and breadcrumb item without reload.

Reply:
sCurrentRecordName is set above (line 586) but not defined. So I guess it exists as an instance/global variable somewheren.

Reply:
Oh, this was an earlier fix attempt. Now we don't need it anymore. Removed it.